server/template/paste.xhtml

```diff
@@ -0,0 +1,61 @@
+<html py:extends="master.xhtml" py:strip="True">
+<body>
+
+  <head py:block="head" py:strip="True">
+    <title py:block="master_title">Paste Logs</title>
+  <style type="text/css">
+   .container {
+       width: 1000px;
+       clear: both;
+   }
+   .container input {
+       width: 100%;
+       clear: both;
+   }
+  </style>
+</head>
+
+<body py:block="body" py:strip="True">
+  <div class="container">
+    <form name="upload_form" action="paste" method="post" enctype="multipart/form-data">
+    <div class="form-group" id="message">
+      <textarea name="message" rows="10" cols="60" placeholder="Enter logs here or upload a text file below" >${raw_logs}</textarea>
+      <br /><br />
+      <input type="file" name="upload" accept="*.txt" ></input>
```

Comment:
It is likely that their log files will have the `.log` file extension. Just allow anything here.